Python/jit.c

```diff
@@ -309,19 +310,19 @@ patch(unsigned char *base, const Stencil *stencil, uint64_t *patches)
                 {
                     unsigned char rd = get_bits(loc32[0], 0, 5);
                     assert(IS_AARCH64_LDR_OR_STR(loc32[1]));
-                    unsigned char rt = get_bits(loc32[1], 0, 5);
-                    unsigned char rn = get_bits(loc32[1], 5, 5);
-                    assert(rd == rn && rn == rt);
+                    // There should be only one register involved:
+                    assert(rd == get_bits(loc32[1], 0, 5));  // rt
```

Comment:
The `// rt` comment is rather cryptic.

Reply:
Renamed `rd` to `reg` and changed the comments.